tests/test-suite.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: testing.kyma-project.io/v1alpha1
+kind: ClusterTestSuite
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: compass-e2e-tests
+spec:
+  maxRetries: 0
```

Comment:
There are sporadic networking issues so we can discuss `maxRetries` option with the rest of the team.

Reply:
The make targets now use tesing.sh and in the test-suite template there maxRetries is set to one.